File: javascript/examples/jacobsthal_numbers.js
var sequences = require('../lib/sequences.js');

let i = 0;
for (let sequenceNumber of sequences.jacobsthalNumberNonRecursiveGenerator(50)) {
    process.stdout.write(`J(${(i++).toString().padStart(2)}): ${sequenceNumber.toString().padEnd(30)}`)
    if (i % 3 == 0) {
        console.log();
    }
}

File: dart/examples/jacobsthal_numbers.dart
import 'dart:io';
import 'package:clounce_math/sequences/jacobsthal_numbers.dart';

void main(List<String> args) async {
  num i = 0;
  await for (var sequenceNumber in JacobsthalNumber.generate(50)) {
    stdout.write('J(' + (i++).toString().padLeft(2) + '): ' + sequenceNumber.toString().padRight(30));
    if (i % 3 == 0) {
      print('');
    }
  }
}